models/bg.model.js

```diff
@@ -69,16 +69,34 @@ exports.addReviewComment = (review_id, reqBody) => {
         }
         return db
             .query(
-            `INSERT INTO comments
+                `INSERT INTO comments
             (author, body, review_id)
             VALUES ($1, $2, $3)
             RETURNING *;`,
                 [username, body, review_id]
             )
-            .then((comment) => { 
+            .then((comment) => {
                 return comment.rows;
             });
     } else {
         return Promise.reject({ status: 400, msg: "Invalid comment" });
     }
 };
+
+exports.updateReviewById = (review_id, inc_votes) => {
+    const votes = inc_votes
+
+    if (!votes) {
+        return Promise.reject({
+            status: 400, msg: 'votes values was incorrect'
+        })
+    }
+    return this.fetchReviewById(review_id)
```

Comment:
Like the use of an existing model to check if something exists first, however it is just an additional call to the db unnecessarily.
PSQL will tell you if the review you're trying to update doesn't exist